doc/application/application.rst

```diff
@@ -341,7 +776,8 @@ entries manually, using the configuration menu is a preferred method.
 
    * Type a numerical value in the round brackets :guilabel:`( )`.
```

Comment:
change "round brackets" to "parentheses"

Reply:
Pre-existing issue, adding a follow-on commit, thanks for the catch